src/user/PreferencesController.ts

```diff
@@ -77,7 +77,7 @@ export class PreferencesController extends BaseController<
       ipfsGateway: 'https://ipfs.io/ipfs/',
       lostIdentities: {},
       selectedAddress: '',
-      useStaticTokenList: false,
+      useTokenDetection: false,
```

Comment:
@Cal-L `useStaticTokenList` used to depict that we are using the static token list instead of the dynamic token list since we have changed the name and the purpose of the flag I feel like the `useTokenDetection` value as `true` will make it less confusing for the condition `token detection ON by default`.